wurst/objects/items/MixingPot/Cure.wurst

```diff
@@ -0,0 +1,38 @@
+package Cure
+
+import PotionsDefinition
+import ID
+import ChannelAbilityPreset
+import ClosureEvents
+import InstantDummyCaster
+import Orders
+import ObjEditingUtils
+
+constant ABIL  = compiletime(ABIL_ID_GEN.next())
```

Comment:
Abilities should be globally registered in a single file to prevent imports just for the IDs.

Reply:
Even for item dummy abilities? I don't see a scenario case where you would need to import it since it will(should) be used only for this item.

Reply:
Well in this case it was the dummy ability that the item uses that was the problem, so testing those directly doesn't seem like a bad ID. Having a single directory of IDs could facilitate having commands to test abilities directly.